### PR TITLE
[Tests-Only]Adjust expected failures file

### PR DESCRIPTION
**Pull request by jasson99**
## Description
This PR adjust expected failures file after the adjustments of `notToImplementOnOcis` tags for the old public webdav API tests.

## Related Issue
https://github.com/owncloud/ocis/issues/2078

## How Has This Been Tested?
- CI

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

**Review by dpakach — COMMENTED** [review]
Looks good
a lot of lines got deleted from the expected failure file..... great work 👍🏼

File: .drone.env [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # The test runner source for API tests
-CORE_COMMITID=b587826832245d9c5f72ab2a9d831b1be6b66210
-CORE_BRANCH=master
+CORE_COMMITID=
+CORE_BRANCH=dontRunOldPublicWebdavInOcis
```
Comment (dpakach) [inline .drone.env:3]: **note** needs this to be changed back to master after https://github.com/owncloud/core/pull/38825#pullrequestreview-681685219 this gets merged

**Review by dpakach — CHANGES_REQUESTED** [review]
just requesting changes to make sure this doesn't gets merged accidently

File: .drone.env (hunk 1 above)
Comment (jasson99) [reply .drone.env]: This is done now.

**Review by phil-davis — APPROVED** [review]
LGTM if CI passes

**Review by dpakach — APPROVED** [review]

**Pull request merged**